src/odemis/acq/stream/_static.py

```diff
@@ -1158,6 +1159,536 @@ def onSpectrumBandwidth(self, value):
 # we could simplify/merge the two stream classes.
 
 
+class TemporalSpectrumStream(StaticStream):
+    """
+    A Temporal spectrum stream which displays only one static image/data.
+
+    """
+
+    def __init__(self, name, image, *args, **kwargs):
+        """
+        name (string)
+        image (model.DataArray(Shadow) of shape (CTYX) or (CT1YX)). The metadata
+        MD_TIME_LIST should be included in order to associate the T to a time value because of the
```

Comment:
It should be possible to load data, which was acquired with "Focus" mode. This means there is not MD_TIME_LIST. Then it makes maybe sense to check accordingly and only display the spectrum and not the time histogram.